docs/swagger.yaml

```diff
@@ -1,54 +1,67 @@
 openapi: 3.0.0
 info:
   title: Author's Haven
-  description: Author's Haven is a Create a community of like minded authors to foster
+  description: Author's Haven is a community of like minded authors to foster
     inspiration and innovation by leveraging the modern web Api on [https://ah-dahlia.herokuapp.com/](https://ah-dahlia-staging.herokuapp.com/)
-  contact:
-    email: you@your-company.com
+
   license:
-    name: Apache 2.0
-    url: http://www.apache.org/licenses/LICENSE-2.0.html
+    name: MIT License
+    url: https://opensource.org/licenses/MIT
   version: 1.0.0
+
+externalDocs:
+  description: Checkout the GitHub repo
+  url: https://github.com/andela/dahlia-ah-backend
+
 servers:
-- url: https://ah-dahlia.herokuapp.com/
-  description: Author's Haven documentation
+- url: /api/v1/
+  description: Local host
+- url: https://ah-dahlia-staging.herokuapp.com/api/v1/
+  description: Heroku deployment
+
+tags:
+- name: Auth
+  description: Operations available for user endpoints
+
 security:
 - ApiKeyAuth: []
 paths:
-  /users:
+  /auth/signup:
```

Comment:
I notice your route is different from the documentation in the readme of this application, kindly check it out

Reply:
Nice catch! I'll jump on it ASAP.